projects/sandbox/export/export/main.py

```diff
@@ -48,32 +53,46 @@ def main(
         architecture:
             A function which takes as input a number of witness
             channels and returns an instantiated torch `Module`
-            which represents a DeepClean network architecture
+            which represents an aframe network architecture
         repository_directory:
             Directory to which to save the models and their
             configs
-        outdir:
+        logdir:
```

Comment:
Given that the logs are the only real output here, I support this name change. However, in that case, let's do away with the `weights is None` case inferring from this directory, since it's meant to be very different from the one where you would ever save model checkpoints. If `weights is None`, just save a randomly initialized model